.travis.yml

```diff
@@ -2,4 +2,12 @@ language: go
 go:
   - 1.4
   - 1.5
+  - 1.7
+  - 1.9
+  - 1.10
```

Comment:
We'll need to put all these version numbers in quotes. See https://github.com/travis-ci/travis-ci/issues/9247 for more information

Reply:
Asking myself: how did I never notice it? Fixed.